src/main/resources/docs/patterns.json

```diff
@@ -176,6 +181,11 @@
     "level" : "Info",
     "category" : "CodeStyle",
     "languages" : [ "Apex" ]
+  }, {
+    "patternId" : "category_apex_errorprone_ApexCSRF",
+    "level" : "Warning",
+    "category" : "ErrorProne",
+    "languages" : [ "Apex" ]
```

Comment:
I think this pattern replaces `category_apex_security_ApexCSRF`. If so, we should set Security category, right?